### DISCUSSION TITLE
[announcement]: alternative to openvsx for self-hosting a marketplace

### DISCUSSION THREAD

**Discussion by ghuntley** [VSCodium/vscodium#1319] [General]
Howdy [vscodium](https://github.com/VSCodium/vscodium) 👋 from Coder!

Today we are open-sourcing https://github.com/coder/code-marketplace which provides a easier way to self-host your own marketplace. See https://coder.com/blog/running-a-private-vs-code-extension-marketplace for more specifics.

**question:** do you want a pull-request with this information added to https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions--marketplace ?

**edit:** done via https://github.com/VSCodium/vscodium/pull/1320

**Comment by daiyam** [comment]
@ghuntley Yeah, a PR would be welcome.

**Comment by daiyam** [comment]
Just for information, I've made the extension [VSIX Manager](https://github.com/zokugun/vscode-vsix-manager) which allows the user to be able to install extensions from different marketplaces.
I plan to add support for github release and more sources.
